README.md

```diff
@@ -1,22 +1,37 @@
 FULLY AUTOMATED REDISTICTING SOFTWARE
 written in java, open source.
 
-CONTRIBUTION GUIDELINES:
```

Comment:
I moved the contrib guidelines further down so that the "HOW TO RUN" section would be first.